addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/dialogs/ZestRecordScriptDialog.java

```diff
@@ -176,6 +186,17 @@ private List<String> getSites() {
         return list;
     }
 
+    private void launchBrowser(String url, String browserName) {
+        ExtensionSelenium extSelenium =
+                Control.getSingleton().getExtensionLoader().getExtension(ExtensionSelenium.class);
+        WebDriver wd = extSelenium.getProxiedBrowserByName(browserName);
```

Comment:
It should handle the exceptions that might happen (e.g. failed to launch the browser) to inform the user and also stop the recording.